pkgs/by-name/twister/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  autoreconfHook,
+  curl,
+  fetchgit,
+  gnunet,
+  jansson,
+  lib,
+  libgcrypt,
+  libmicrohttpd,
+  libnfc,
```

Comment:
This looks like an unused input.

Other than that, LGTM.

Reply:
Thanks! Fixed.